server/notebooks/views.py

```diff
@@ -53,6 +54,14 @@ def notebook_revisions(request, pk):
         'title': nb.title,
         'notebookId': nb.id
     }
+    files = [
+        {'filename': file.filename, 
+        'id': file.id, 
+        'last_updated': file.last_updated.isoformat(sep=' '),
+        'size': len(file.content)}
```

Comment:
This is probably going to be horribly slow if the file is at all big... we're probably going to move to a better/different storage solution sooner than later but for now please at least add a `FIXME` comment about how this is inefficient.